.github/workflows/autogpt-builder-ci.yml

```
@@ -0,0 +1,34 @@
+name: AutoGPT Builder CI
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'rnd/autogpt_builder/**'
+  pull_request:
+    paths:
+      - 'rnd/autogpt_builder/**'
```

Sorry for not mentioning this earlier, but the trigger filter should probably also include this workflow file:
```suggestion
    paths:
      - '.github/workflows/autogpt-builder-ci.yml'
      - 'rnd/autogpt_builder/**'
  pull_request:
    paths:
      - '.github/workflows/autogpt-builder-ci.yml'
      - 'rnd/autogpt_builder/**'
```